Note that the pkgconfigdir line has been changed by USES=pathfix.

--- Makefile.am.orig	2020-07-25 08:25:49 UTC
+++ Makefile.am
@@ -6,7 +6,7 @@ pkgdata_DATA = bash_completion
 compatdir = $(sysconfdir)/bash_completion.d
 compat_DATA =
 
-profiledir = $(sysconfdir)/profile.d
+profiledir = $(prefix)/share/bash-completion
 profile_DATA = bash_completion.sh
 
 pkgconfigdir = $(prefix)/libdata/pkgconfig
